docs/components/search-agent.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: docs
+title: Search Agent
+description: Documentation for the correct implementation of the Search Agent.
+group: components
+---
+
+Small and adaptive tag for adding context to just about any content.
+
+## Contents
+
+* Will be replaced with the ToC, excluding the "Contents" header
+{:toc}
+
+## Example
+
+Default Search Agent in horizontal alignment
+
+{% example html %}
+<div class="card">
+  <div class="card-block search-agent">
+    <div class="row">
+      <div class="col-12 col-sm-6 push-sm-6">
+        <div class="search-agent-body">
+          <p class="search-agent-subtitle">Suchagent</p>
+          <h4 class="search-agent-title">Direkt in Deinen Posteingang</h4>
+          <p class="search-agent-text">Sobald ein neuer Job eingestellt wird, schicken wir Dir eine Mail.</p>
+          <a href="#" class="btn btn-primary">Benachrichtigung anlegen</a>
```

Comment:
The intent here is not to link to another page but rather trigger an action like showing a modal so the user only has to enter his e-mail address and be done and happy :)